README.md

```diff
@@ -1,3 +1,6 @@
+# This Is My CEG-4110 Spring 2020 Project
+**by Erik M. Buck**
+
 ## Craft
```

Comment:
Should this really be a second level heading or should it be a first level heading